go-controller/pkg/kubevirt/dhcp.go

```diff
@@ -92,7 +93,7 @@ func retrieveDNSServiceClusterIPs(k8scli *factory.WatchFactory) (string, string,
 	return clusterIPv4, clusterIPv6, nil
 }
 
-func composeDHCPv4Options(cidr, dnsServer, controllerName, namespace, vmName string) *nbdb.DHCPOptions {
+func ComposeDHCPv4Options(cidr, dnsServer, controllerName, namespace, vmName string) *nbdb.DHCPOptions {
```

Comment:
There is a lot of stuff going on there (the external ids, the timeout lease time the ARP proxy thing) I prefer to re-use them since the kubevirt_test is semi integration that do not really care about specifics.